### PR TITLE
[ZEPPELIN-3701].Missing first several '0' and losing digital accuracy in result table

### PR DESCRIPTION
**Pull request by Deegue**
### What is this PR for?

Improvements:
-Datas like '00058806' will be displayed correctly instead of '58806'.
-Datas like '5880658806' will be displayed correctly instead of '5.880659E9'.

### What type of PR is it?
[Refactoring]

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3701

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

Author: heguozi <zyzzxycj@gmail.com>

### CODE REVIEW

File: zeppelin-web/src/app/tabledata/tabledata.js [hunk 1]
```diff
@@ -60,8 +60,10 @@ export default class TableData extends Dataset {
           columnNames.push({name: col, index: j, aggr: 'sum'});
         } else {
           let valueOfCol;
-          if (!isNaN(valueOfCol = parseFloat(col)) && isFinite(col)) {
-            col = valueOfCol;
+          if (!(col[0] === '0' || col.length >= 7)) {
```
[the remark this answers is not in the document]
Comment (Deegue) [reply zeppelin-web/src/app/tabledata/tabledata.js]: yes, thank you for your advice!

[the remark this answers is not in the document]
Comment (Deegue) [reply zeppelin-web/src/app/tabledata/tabledata.js]: @felixcheung It seems to have the same problem 'Failed while enforcing releasability'.
I would very appreciate it if you could tell me how to resolve this problem.
CI : https://travis-ci.org/Deegue/zeppelin/builds/415278918

**Comment by Deegue** [comment]
@Savalek It won't be affected in this case , it will keep '00987...' and '...'(length>16) as String .
Other cases are still the same as before.

**Comment by Savalek** [comment]
I mean, it would be nice to add a check `if col[0] === '+'` to ` if (!(col[0] === '0' || col.length > float64MaxDigits)) {`

**Comment by Deegue** [comment]
That's the problem I haven't met . The check added , but I have trouble in CI tests .
I'd appreciate it if you could tell me how to resolve the problem : CI : https://travis-ci.org/Deegue/zeppelin/builds/415278918

**Comment by Deegue** [comment]
@felixcheung CI passed : https://travis-ci.org/Deegue/zeppelin/builds/415389747

**Comment by felixcheung** [comment]
merging if no more comment

**Comment by mmukherjee** [comment]
Thanks folks for helping with this. Can you please let me know by when would this fix be rolled out as as tagged release? Is there a release roadmap I could refer to?

**Comment by felixcheung** [comment]
this should be in 0.9. as for release date plan, please ask on dev@zeppelin.apache.org

**Pull request closed without merging**